vitest.config.ts

```diff
@@ -24,6 +24,7 @@ export default defineConfig({
       include: [
         'src/**/*.jsx',
         'src/**/*.tsx',
+        'src/**/*.ts',
```

Comment:
I think we need to exclude `*.styles.ts` from the coverage results because we don't write tests for them

Reply:
Thank you for your attention! Excluded *.styles.ts files from the coverage results.